libPhoneNumberGeocoding/NBPhoneNumberOfflineGeocoder.m

```diff
@@ -0,0 +1,169 @@
+//
+//  NBPhoneNumberOfflineGeocoder.m
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import "NBPhoneNumberOfflineGeocoder.h"
+#import "Metadata/NBGeocoderMetadataHelper.h"
+#import "NBPhoneNumber.h"
+#import "NBPhoneNumberUtil.h"
+
+@implementation NBPhoneNumberOfflineGeocoder {
+ @private
+  NBPhoneNumberUtil *_phoneNumberUtil;
+  NSCache<NSString *, NBGeocoderMetadataHelper *> *_metadataHelpers;
+  NSNumber const *_US_REGION_CODE;
```

Comment:
Can we make this a static const? At least, make it `static const NSInteger US_REGION_CODE` and we can box it into NSNumber when we need them.

Reply:
Ended up being an unnecessary constant. removed.